posts/2024-07-20-rladies-bioconductor-collaboration/index.qmd

```diff
@@ -1,13 +1,14 @@
 ---
-title: "Invitation to participate in the new R-Ladies–Bioconductor collaboration!"
+title: "New R-Ladies–Bioconductor collaboration!"
 description: |
   Announcing a new collaboration between R-Ladies and Bioconductor
 author:
   - name: Janani Ravi
     url: https://github.com/jananiravi
-date: 2024-07-20
+date: 2024-07-20 | Updated: 2025-02-06
```

Comment:
@jananiravi are you able to check locally that using two dates separated by a | will work correctly (I'm not sure that it will, also Quarto blog has a page that lists all posts by date: https://blog.bioconductor.org/ so I don't know how it will handle that)? If not, I can check it when I get a chance, I'm at a conference now till Sat.